src/object-registry.js

```diff
@@ -1,12 +1,20 @@
 import * as tf from '@tensorflow/tfjs-core';
 
+/**
+ * ObjectRegistry stores a map of {id -> object} and a map of {id -> gc},
+    where gc denotes if we want the object of a given id to be garbage collected
+ *
+ * @property {Object.<string, Plan>} objects
+ * @property {Object.<string, Protocol>} gc
+ */
 export default class ObjectRegistry {
   constructor() {
     this.objects = {};
     this.gc = {};
   }
 
   set(id, obj, gc = false) {
+    // We want to get rid of the current id and object in the objects map before replacing them
```

Comment:
Personally I'd start with an imperative sentence for one-line comments, like "Remove the current id and object..."